src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -139,25 +139,6 @@ def test_edit_slug_valid(self):
         doc = pq(response.content)
         assert doc('form').attr('action') != old_edit
 
-    def test_edit_summary_escaping(self):
```

Comment:
This test was a bit redundant, there is already an `xss` test below in the same file, and the test I fixed regarding the widget in `translations/tests/test_widgets.py`